dhis-2/dhis-api/src/main/java/org/hisp/dhis/calendar/impl/NepaliCalendar.java

```diff
@@ -475,6 +500,39 @@ private DateInterval toDayIsoInterval(DateTimeUnit dateTimeUnit, int offset, int
   private static final Map<Integer, int[]> CONVERSION_MAP = new HashMap<>();
```

Comment:
Can you add a comment, so we know until which year this is valid?
I belive that at some point we need to add more elements to the map...?